### DISCUSSION TITLE
Python Debugger Extension: Class Information and Set a Breakpoint at All Class Methods

### DISCUSSION THREAD

**Discussion by rheniser** [microsoft/vscode-discussions#1086] [Extension Development QnA]
I hope this is the right place to ask this. I am curious whether it is possible to do the following?

Will it be possible to get class information for the methods for which we set breakpoints?

Will it be possible to get the ability to set a breakpoint at all methods in a class?

I opened a request: https://github.com/microsoft/vscode-python-debugger/issues/254

Just curious if this is even possible--Should I get my hope up?

**Comment by burkeholland** [comment]
Closing in favor of the linked issue.
